logstash-core-event-java/src/main/java/com/logstash/Event.java

```diff
@@ -28,9 +28,9 @@
     public static final String VERSION = "@version";
     public static final String VERSION_ONE = "1";
 
-    private static final ObjectMapper mapper = new ObjectMapper();
-
-    // TODO: add metadata support
+    private static final Logger DEFAULT_LOGGER = new StdioLogger();
+    private transient final ObjectMapper mapper = new ObjectMapper();
+    private transient static Logger logger = DEFAULT_LOGGER;
```

Comment:
I'm OK with a global variable here if we comment that it's intended for testing/specs.